### ISSUE TITLE
Blending mode and LUT changes not updating properly

### ISSUE THREAD

**Issue by smlpt** [scenerygraphics/sciview#441]
After loading either the SciView Volume demo, the volume timeseries demo or the T1 head from Fiji as a volume, the first blend mode change is applied properly, but all subsequent changes are not. Similarly with the LUTs that won't be applied straightaway, but only after toggling visibility on and off. The LUT preview bar also does not update properly and sometimes only after selecting the next LUT. With the T1 head, the blend mode would sometimes get completely stuck and wouldn't change at all. 
I haven't found a specific way to exactly reproduce it yet, but something about the blend mode and LUT changes doesn't work like its supposed to do.
Can anyone reproduce the issue on their end?

**Comment by kephale** [comment]
I'm seeing this with the blending mode. I've also seen issues with color map updates being delayed in the past. Currently LUTs seem to be fine for me.

**Comment by kephale** [comment]
Documenting an example of the LUT preview not being initialized after running VolumeRenderDemo:
<img width="1262" alt="image" src="https://github.com/scenerygraphics/sciview/assets/400105/429419f9-30f6-4586-9c50-7be9ab328011">

**Comment by kephale** [comment]
Ok, so I can verify that the `node.renderingMethod` is being correctly updated for the Volume, but I cannot see a visual change between any of the renderingMethods. I need to call in @skalarproduktraum @moreApi for that issue.

**Comment by kephale** [comment]
Multiple metadata keys have been introduced for colormaps it seems:
metadata["sciview.colormap-name"]
metadata["sciviewColormap"]

**Comment by smlpt** [comment]
@kephale This was merged into master with #499 right? Because I just pulled the master and the LUTs still don't update. Same with the blending mode. Can you reproduce this?

**Comment by kephale** [comment]
I get this with the Volume Render demo:
<img width="1184" alt="image" src="https://github.com/scenerygraphics/sciview/assets/400105/81c95d60-9f08-4c29-a975-6af7267f3933">

Changing the LUT works. Something seems to be funky with rendering still though.

**Comment by kephale** [comment]
I'm not sure this is on the sciview side for the rendering mode update. 

<img width="1176" alt="image" src="https://github.com/scenerygraphics/sciview/assets/400105/d80c3dda-594e-4f05-8ed6-6b187db04793">

It looks like the `renderingMode` attribute is changing when the UI is updated. When I update the scene, that also does not trigger the `renderingMode` to change.

@skalarproduktraum @moreApi is this an upstream issue?

**Comment by skalarproduktraum** [comment]
This _is_ indeed an upstream issue, as there's actually only the alpha blending mode implemented 😅 Maybe remove for the moment?
